src/main/java/nl/_42/max_login_attempts_spring_boot_starter/service/Attempt.java

```diff
@@ -0,0 +1,46 @@
+package nl._42.max_login_attempts_spring_boot_starter.service;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+
+public class Attempt {
```

Comment:
Deze class zou volgens mij beter `UserAttemptsMonitor` kunnen heten. Of `AttemptsMonitor`

Reply:
AttemptsMonitor klinkt dan wel beter